src/sentry/static/sentry/app/components/events/contexts/trace/types.tsx

```diff
@@ -0,0 +1,16 @@
+export enum TraceKnownDataType {
+  TRACE_ID = 'trace_id',
+  SPAN_ID = 'span_id',
+  PARENT_SPAN_ID = 'parent_span_id',
+  OP_NAME = 'op',
+  STATUS = 'status',
+  TRANSACTION_NAME = 'transaction_name',
+}
+
+export type TraceKnownData = {
+  trace_id?: string;
+  span_id?: string;
+  parent_span_id?: string;
+  op?: string;
+  status?: string;
```

Comment:
Are we able to leverage the `TraceContext` type in `app/types/index` here?

Reply:
should be fixed in https://github.com/getsentry/sentry/pull/19078/commits/031336be0f4a9012d7497c37d320b50d571c086c